File: articles/bayesian-computation-and-updating-beliefs/cpp/bayesian_update.cpp
#include <iostream>
#include <iomanip>
int main() { double post_alpha=2.0+113.0, post_beta=2.0+72.0; std::cout << std::fixed << std::setprecision(6) << "posterior_mean=" << post_alpha/(post_alpha+post_beta) << "\n"; return 0; }

File: articles/bayesian-computation-and-updating-beliefs/fortran/bayesian_update.f90
program bayesian_update
  implicit none
  real :: post_alpha, post_beta
  post_alpha = 2.0 + 113.0
  post_beta = 2.0 + 72.0
  print *, "posterior_mean=", post_alpha / (post_alpha + post_beta)
end program bayesian_update
